src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -236,6 +236,7 @@ public static class Agent
                 public static readonly string AllowAllEndpoints = "agent.allowAllEndpoints"; // remove after sprint 120 or so.
                 public static readonly string AllowAllSecureFiles = "agent.allowAllSecureFiles"; // remove after sprint 121 or so.
                 public static readonly string BuildDirectory = "agent.builddirectory";
+                public static readonly string Diagnostic = "agent.diagnostic";
```

Comment:
should we call this system.diag? since the other one is called system.debug

Reply:
We went back and forth on this a couple times, we can discuss with Bryan but we had landed on agent last time we talked.